securedrop/tests/test_source.py

```diff
@@ -318,3 +318,24 @@ def test_login_with_overly_long_codename(self, mock_hash_codename):
             self.assertFalse(mock_hash_codename.called,
                              "Called hash_codename for codename w/ invalid "
                              "length")
+
+    @patch('source.app.logger.warning')
+    def test_generate_unique_codename(self, mock_warning):
+        codename = source.generate_unique_codename()
+        self.assertGreater(len(codename), 0)
+        mock_warning.assert_not_called()
```

Comment:
My intent was to run the code path used most of the time and assert it does not display any warning. This should never fail and if it does, we know there is a regression. If you still think this is not worth testing I'll remove it.